src/elements/alert/alert/readme.md

```diff
@@ -13,6 +13,7 @@ The component can optionally display a `sbb-icon` at the component start using t
   Between Bern and Olten from 03.11.2021 to 05.12.2022 each time from 22:30 to 06:00 o'clock
   construction work will take place. You have to expect changed travel times and changed
   connections.
+  <sbb-link href="https://www.sbb.ch">Find out more</sbb-link>
```

Comment:
Can't add a comment on line 8 as i would, however, i would add something like:
'the unnamed slot can also contain one or more `sbb-link`.....`